internal_packages/keybase/stylesheets/main.less

```diff
@@ -0,0 +1,88 @@
+@import "ui-variables";
+@import "ui-mixins";
+
+.keybase-profile {
+  border: 1px solid black;
```

Comment:
Yeah, this is definitely a placeholder until we put something not-terrible in place. :)